googletest/src/gtest.cc

```diff
@@ -564,6 +606,8 @@ void TypeParameterizedTestSuiteRegistry::CheckForInstantiations() {
         "UninstantiatedTypeParameterizedTestSuite<" + testcase.first + ">";
     RegisterTest(  //
         "GoogleTestVerification", full_name.c_str(),
+        'S',      // Default size parameter
```

Comment:
I need a bit of clarification here. So this is creating a `S` size param test by default? And all the macros for creating tests have a default `X` assigned? How are these working together, I'm getting lost in this a bit.

Reply:
I was confused by this too. This is a test for Google test and doesn't matter what the size is.

To use the size and tags a user would need to use the internal macros or define new macros. If a user doesn't do that all the tests get a size of `X` and tag of `ALL`. The default when running is size = `*` and tag = `*`

Reply:
Got it, thanks!